src/main/java/org/kohsuke/github/GHPerson.java

```diff
@@ -178,22 +173,18 @@ public GHRepository getRepository(String name) throws IOException {
      * @return the gravatar id
      * @deprecated No longer available in the v3 API.
      */
+    @Deprecated
     public String getGravatarId() {
-        return gravatar_id;
+        return "";
     }
 
     /**
-     * Returns a string like 'https://secure.gravatar.com/avatar/0cb9832a01c22c083390f3c5dcb64105' that indicates the
-     * avatar image URL.
+     * Returns a string of the avatar image URL.
      *
      * @return the avatar url
      */
     public String getAvatarUrl() {
-        if (avatar_url != null)
```

Comment:
If this is null, it is still more meaningful that the `""` returned from gravatar.